dot/parachain/availability-store/availability_store.go

```diff
@@ -420,89 +410,87 @@ func branchesFromChunks(chunks [][]byte) (branches, error) {
 }
 
 // Run runs the availability store subsystem
-func (av *AvailabilityStoreSubsystem) Run(ctx context.Context, OverseerToSubsystem chan any,
-	SubsystemToOverseer chan any) {
-
-	av.wg.Add(1)
-	go av.processMessages()
+func (av *AvailabilityStoreSubsystem) Run(
+	ctx context.Context, OverseerToSubsystem chan any, SubsystemToOverseer chan any,
```

Comment:
since OverseerToSubsystem will be only readed from and SubsystemToOverseer will be only written to, would it make sense to pass them as directed channels arguments?

Reply:
updated